backend/organisation/src/main/java/org/egov/service/NotificationService.java

```diff
@@ -0,0 +1,274 @@
+        List<String> mobileNumbers = organisation.getContactDetails().stream().map(contactDetails -> contactDetails.getContactMobileNumber()).collect(Collectors.toList());
+        String CBOUrl = getShortnerURL(config.getCboUrlHost() + config.getCboUrlEndpoint());
+
+        Map<String, List<String>> smsDetails = new HashMap<>();
+
+        smsDetails.put("orgNames", Collections.singletonList(orgName));
+        smsDetails.put("personNames", personNames);
+        smsDetails.put("mobileNumbers", mobileNumbers);
+        smsDetails.put("CBOUrl", Collections.singletonList(CBOUrl));
+
+
+        return smsDetails;
+    }
+
+    private Map<String, String> getSMSDetailsForUpdate(OrgRequest request) {
+
+        RequestInfo requestInfo= request.getRequestInfo();
+        Organisation organisation=request.getOrganisations().get(0);
+        String uuid=organisation.getAuditDetails().getLastModifiedBy();
+        String orgName = organisation.getName();
+        String tenantId=organisation.getTenantId();
+        String CBOUrl = getShortnerURL(config.getCboUrlHost() + config.getCboUrlEndpoint());
+
+
+        //get name, mobileNumber from hrms
+        log.info("get name, mobileNumber from hrms");
+        Map<String , String> employeeDetails=hrmsUtils.getEmployeeDetailsByUuid(requestInfo,tenantId,uuid);
+
+        Map<String, String> smsDetails = new HashMap<>();
+
+        smsDetails.put("orgNames", orgName);
+        smsDetails.put("personName", employeeDetails.get("userName"));
+        smsDetails.put("mobileNumber", employeeDetails.get("mobileNumber"));
+        smsDetails.put("CBOUrl", CBOUrl);
+
+
+        return smsDetails;
+    }
+
+    private String getMessage(OrgRequest request, boolean isCreateOperation) {
+        String message = null;
+
+        if (isCreateOperation) {
+            message = getMessage(request, ORGANISATION_CREATE_LOCALIZATION_CODE);
+        } else {
+            message = getMessage(request, ORGANISATION_UPDATE_LOCALIZATION_CODE);
+        }
+
+        return message;
+    }
+
+    /**
+     * Gets the message from localization
+     *
+     * @param request
+     * @param msgCode
+     * @return
+     */
+    public String getMessage(OrgRequest request, String msgCode) {
+        String rootTenantId = request.getOrganisations().get(0).getTenantId().split("\\.")[0];
+        RequestInfo requestInfo = request.getRequestInfo();
+        Map<String, Map<String, String>> localizedMessageMap = getLocalisedMessages(requestInfo, rootTenantId,
+                OrganisationConstant.ORGANISATION_NOTIFICATION_ENG_LOCALE_CODE, OrganisationConstant.ORGANISATION_MODULE_CODE);
+        return localizedMessageMap.get(OrganisationConstant.ORGANISATION_NOTIFICATION_ENG_LOCALE_CODE + "|" + rootTenantId).get(msgCode);
+    }
+
+    /**
+     * Builds msg based on the format
+     *
+     * @param message
+     * @param userDetailsForSMS
+     * @return
+     */
```

Comment:
Declare constants for below type of Strings.
"{individualName}"

Reply:
Keeping it generic for now